roles/postgresql/templates/postgresql.container.j2

```diff
@@ -0,0 +1,11 @@
+[Container]
+ContainerName={{ postgresql_container_name }}
+Image={{ postgresql_image }}
+Network=host
+Volume={{ postgresql_data_dir }}:/var/lib/pgsql/data:Z
+Volume={{ playbook_dir }}/postgresql-init:/opt/app-root/src/postgresql-init:Z
```

Comment:
I'm not a fan of mounting something from the playbook directory. In general I'd assume you can delete the "installer" and this would break that assumption. We've already made that mistake in the past (looking at `/root/ssl-build`) and I'd prefer to avoid it.